### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by Jamesidkk** [maximmax42/Discord-CustomRP#190] [Q&A] [answered]
Hello Max, could this application work on Windows 7?

**Comment by maximmax42** [comment, marked as answer]
1.16.12 still supports Win7, newer versions won't because analytics library dropped support for Win7.

**Reply by Jamesidkk** [reply]
Uhh then how do I download that version (sorry I'm dumb)

**Reply by maximmax42** [reply]
https://github.com/maximmax42/Discord-CustomRP/releases/tag/1.16.12

**Reply by Jamesidkk** [reply]
Tks man ur the best 😁😁

**Reply by maximmax42** [reply]
Actually, would you mind doing a little experiment for me?

**Reply by Jamesidkk** [reply]
Uhh sure man

**Reply by maximmax42** [reply]
Download latest CustomRP version as a zip (https://github.com/maximmax42/Discord-CustomRP/releases/download/1.16.13/CustomRP.1.16.13.zip), unpack it somewhere, then replace Microsoft.AppCenter.dll, Microsoft.AppCenter.Analytics.dll and Microsoft.AppCenter.Crashes.dll with the files from 1.16.12 (https://github.com/maximmax42/Discord-CustomRP/releases/download/1.16.12/CustomRP.1.16.12.zip) and check if the app will work after that.

**Reply by Jamesidkk** [reply]
Ok Ill try it now

**Reply by Jamesidkk** [reply]
Wow it works lol

**Reply by maximmax42** [reply]
Yeah, I've already implemented that in the installer. So you can install newest versions without doing all that.

**Reply by Jamesidkk** [reply]
Oh okay nice tks man👍
